quippy/quippy/gap_tools.py

```diff
@@ -0,0 +1,274 @@
+import numpy as np
+from xml.etree.ElementTree import parse, fromstring, tostring, ElementTree
+import re
+import os
+from quippy.descriptors import Descriptor
+from quippy.potential import Potential
+from ase.data import chemical_symbols
+from scipy.stats import norm
```

Comment:
scipy would be a new dependency for quippy. OK if really needed, but could we get away with numpy only? It has a multivariate normal random function, but probably not a triangular solve.

Reply:
If we do keep `scipy` it should be added to `quippy/setup.py`

Reply:
I could use just a regular numpy.solve? Would obviously be slower than an explicit triangular

Reply:
Maybe a compromise is to try to import scipy, but catch the ImportError if this fails. If scipy is available can then use the faster triangular solve, falling back to numpy.linalg.solve if it's not.